Makefile

```diff
@@ -84,7 +84,7 @@ clean:
 	rm -f $(install_dir)/firrtl.jar
 	rm -f $(install_dir)/firrtl
 	rm -f $(install_dir)/firrtl-stanza
-	"$(sbt)" "clean"
+	$(SBT) "clean"
```

Comment:
@ucbjrl, if you need to point to paths with spaces, can't you include the quotes within the definition of environment variable SBT, such as SBT = "/path/with spaces"?